src/main/java/com/google/firebase/auth/AbstractFirebaseAuth.java

```diff
@@ -1058,6 +1060,78 @@ protected OidcProviderConfig execute() throws FirebaseAuthException {
     };
   }
 
+  /**
+   * Gets a page of OIDC Auth provider configs starting from the specified {@code pageToken}.
+   *
+   * @param pageToken A non-empty page token string, or null to retrieve the first page of provider
+   *     configs.
+   * @param maxResults Maximum number of provider configs to include in the returned page. This may
+   *     not exceed 100.
+   * @return A {@link ListProviderConfigsPage} instance.
+   * @throws IllegalArgumentException If the specified page token is empty, or max results value is
+   *     invalid.
+   * @throws FirebaseAuthException If an error occurs while retrieving user data.
+   */
+  public ListProviderConfigsPage<OidcProviderConfig> listOidcProviderConfigs(
+        @Nullable String pageToken, int maxResults) throws FirebaseAuthException {
+    return listOidcProviderConfigsOp(pageToken, maxResults).call();
+  }
+
+  /**
```

Comment:
Mention the default maxResults in the docs.

Reply:
Done. I followed the same pattern as in the list user methods.